connectors/sources/microsoft_teams.py

```diff
@@ -281,6 +720,475 @@ async def ping(self):
+                "attendees": attendee_list,
+                "online_meeting_url": item["onlineMeeting"].get("joinUrl")
+                if item.get("onlineMeeting")
+                else "",
+                "description": item.get("bodyPreview"),
+                "meeting_detail": self.get_calendar_detail(calendar=item),
+                "location": [
+                    f"{location['displayName']}" for location in item["locations"]
+                ]
+                if item.get("locations")
+                else [],
+            }
+        )
+        self.map_document_with_schema(
+            document=document, item=item, document_type=self.schema.meeting
+        )
+        return document
+
+    def format_channel_message(self, item, channel_name):
+        document = {"type": TeamEndpointName.MESSAGE.value}
+        document.update(
+            {  # pyright: ignore
+                "sender": item["from"]["user"].get("displayName")
+                if item.get("from") and item["from"].get("user")
+                else "",
+                "channel": channel_name,
+                "message": html_to_text(html=item.get("body", {}).get("content")),
+                "attached_documents": self.get_attachment_names(
+                    attachments=item["attachments"]
+                )
+                if item.get("attachments")
+                else "",
+            }
+        )
+        self.map_document_with_schema(
+            document=document, item=item, document_type=self.schema.channel_message
+        )
+        return document
+
+    def format_channel_meeting(self, reply):
+        document = {"type": TeamEndpointName.MEETING.value}
+        event = reply["eventDetail"]
+        if event.get("@odata.type") == "#microsoft.graph.callEndedEventMessageDetail":
+            particpant_list = []
+            for particpant in event.get("callParticipants", []):
+                user = particpant.get("participant", {}).get("user")
+                if user:
+                    particpant_list.append(user.get("displayName"))
+            particpant_names = ", ".join(particpant_list)
+            document.update(
+                {
+                    "_id": event.get("callId"),
+                    "_timestamp": reply.get("lastModifiedDateTime"),
+                    "particpants": particpant_names,
+                }
+            )
+        elif (
+            event.get("@odata.type")
+            == "#microsoft.graph.callRecordingEventMessageDetail"
+        ):
+            if event.get("callRecordingUrl") and (
+                ".sharepoint.com" in event["callRecordingUrl"]
+            ):
+                document.update(
+                    {
+                        "title": event.get("callRecordingDisplayName"),
+                        "recording_url": event.get("callRecordingUrl"),
+                    }
+                )
+        return document
+
+    async def get_user_chat_messages(self, chat, message):
```

Comment:
To avoid confusion API methods

```suggestion
    async def format_user_chat_messages(self, chat, message):
```